### ISSUE TITLE
Unable to load cabal files for snapshot

### ISSUE THREAD

**Issue by giuliohome** [commercialhaskell/stackage-nightly#4]
I'm trying to run `stack install cabal-install` as per [these instructions](https://github.com/haskell/haskell-ide-engine#installation-with-stack) and I get the following error

```
$ stack install cabal-install
Unable to load cabal files for snapshot

----
Deleting cached snapshot file: C:\sr\build-plan\nightly-2018-09-28.yaml
Recommendation: try running again. If this fails again, open an upstream issue at:
https://github.com/fpco/stackage-nightly/issues/new
----

Unable to parse cabal file for bhoogle-0.1.3.5@sha256:a3393794b22faabeb564c57f4a9506390b6b97b9792c6b4e130f15bf116099fd,1806: NoParse "license" 7

```

**Comment by snoyberg** [comment]
Please run stack upgrade.

**Comment by giuliohome** [comment]
This is the output
```
MyHome@MyHome-PC MINGW64 /C/haskell/ide/haskell-ide-engine (master)
$ stack upgrade
Current Stack version: 1.6.3, available download version: 1.7.1
Newer version detected, downloading
Querying for archive location for platform: windows-x86_64-static
Querying for archive location for platform: windows-x86_64
Downloading from: https://github.com/commercialhaskell/stack/releases/download/v1.7.1/stack-1.7.1-windows-x86_64.tar.gz
Download complete, testing executable
Version 1.7.1, Git revision 681c800873816c022739ca7ed14755e85a579565 (5807 commits) x86_64 hpack-0.28.2
New stack executable available at C:\Users\MyHome\AppData\Roaming\local\bin\stack.exe
```
Now if I do `stack install` I get 
```
Stack looks for packages in the directories configured in the 'packages' and 'extra-deps' fields defined in your stack.yaml
The current entry points to C:\haskell\ide\haskell-ide-engine\submodules\brittany\ but no .cabal or package.yaml file could be found there.
```
